README.md

```diff
@@ -12,7 +12,7 @@
 
 The scroblr extension adds scrobbling capabilities to the following websites:
 
-Abacast, AccuRadio, Amazon Music, Bandcamp, Beats Music, Digitally Imported, Dubtrack.fm, Earbits, Focus@Will, Google Play, Hoopla, Jango, NPR, Pandora, Player.fm, plug.dj, Pocket Casts, Relevant Magazine, Rhapsody, SomaFM, Songza, TheDrop.club, This Is My Jam, Tidal, VK, YouTube
+Abacast, AccuRadio, Amazon Music, Bandcamp, Beats Music, Digitally Imported, Dubtrack.fm, Earbits, Focus@Will, Google Play, Hoopla, Jango, NPR, Pandora, Player.fm, plug.dj, Pocket Casts (and Beta version), Relevant Magazine, Rhapsody, SomaFM, Songza, TheDrop.club, This Is My Jam, Tidal, VK, YouTube
```

Comment:
I don't think the "and Beta version" is necessary, can just omit it and leave this file as it.

Reply:
Done